applications/nrf5340_audio/README.rst

```diff
@@ -352,6 +352,211 @@ The following figure shows the back of the development kit without the case:
 
 For the description of the relevant PCB elements, see the `User interface`_ section.
 
+.. _solderbridge_overview:
+
+Solderbridge overview
+---------------------
+
```

Comment:
We need an introductory line to each table we add.
My suggestion:
```suggestion

The nRF5340 Audio DK has a range of solder bridges for enabling or disabling selected functionalities. Changes to these are not needed for normal use of the DK.

The following table is a complete overview of the solder bridges on the nRF5340 Audio DK.

```

Reply:
updated